src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
-body {
+@import 'variables';
+@import 'mixin';
+
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include center_the_element;
+}
+
+.stopwatch {
+  width: $stopwatch_size;
+  height: $stopwatch_size;
```

Comment:
Variables should be named in a way that is consistent with their purpose and should follow the `$block-name__element-name--modifier-name` format. Check if `$stopwatch_size` follows this naming convention and represents its purpose accurately.